Makefile

```diff
@@ -36,15 +38,19 @@ ifdef NEON64_CFLAGS
 endif
 ifdef SSSE3_CFLAGS
   HAVE_SSSE3 = 1
+  HAVE_FAST_UNALIGNED_ACCESS = 1
 endif
 ifdef SSE41_CFLAGS
   HAVE_SSE41 = 1
+  HAVE_FAST_UNALIGNED_ACCESS = 1
 endif
 ifdef SSE42_CFLAGS
   HAVE_SSE42 = 1
+  HAVE_FAST_UNALIGNED_ACCESS = 1
```

Comment:
You say in your commit that SSE4.1 and SSE4.2 are now useless. Why are they still part of the Makefile?